src/components/ReservationForm.js

```diff
@@ -50,36 +50,34 @@ const ReservationForm = () => {
   };
 
   return (
-    <div className="w-3/4 flex flex-col items-center">
+    <div className="w-3/4 flex flex-col py-4 items-center">
       { doctorId !== undefined && (
-        <div>
+        <div className="flex flex-col items-center pb-4 gap-2">
           <p>
-            Book an appointment with Dr.
+            Book an appointment with M.D.
             {' '}
```

Comment:
Try to delete the blank spaces, that generate the esilint, this is a bad practice

Reply:
Done! 😄